src/Components/Components/src/Rendering/ComponentState.cs

```diff
@@ -36,8 +36,8 @@ public ComponentState(Renderer renderer, int componentId, IComponent component,
             Component = component ?? throw new ArgumentNullException(nameof(component));
             _renderer = renderer ?? throw new ArgumentNullException(nameof(renderer));
             _cascadingParameters = CascadingParameterState.FindCascadingParameters(this);
-            CurrrentRenderTree = new RenderTreeBuilder(renderer);
-            _renderTreeBuilderPrevious = new RenderTreeBuilder(renderer);
+            CurrrentRenderTree = new RenderTreeBuilder();
```

Comment:
`CurrentRenderTree` instead of `CurrrentRenderTree `

Reply:
Thanks!